### ISSUE TITLE
LaTeX - \underbrace nor \overbrace are rendering properly

### ISSUE THREAD

**Issue by mattiasJohnson** [Requarks/wiki#1566]
**Describe the bug**
The LaTeX commands `\underbrace{a}_{b}`, `\overbrace{c}^{d}` are not rendering properly. They succesfully place the expression under (or over) but the braces do not render.

**To Reproduce**
1. Insert the LaTeX code
```
$$
\underbrace{a}_{b}
$$
```
respectively
```
$$
\overbrace{c}^{d}
$$
```
2. Render page and observe that there are no brackets.

**Expected behavior**

---

```
\underbrace{a}_{b}
```
Succesful rendering from KaTeX.org:
![image](https://user-images.githubusercontent.com/5561521/75978647-42397080-5edf-11ea-83e3-30696eb14013.png)

Failed rendering in wiki.js:
![image](https://user-images.githubusercontent.com/5561521/75978714-6dbc5b00-5edf-11ea-97f6-38323e9f772e.png)

---

```
\overbrace{c}^{d}
```

Succesful rendering from KaTeX.org:
![image](https://user-images.githubusercontent.com/5561521/75978823-a3614400-5edf-11ea-969d-4832ff93b3b0.png)

Failed rendering in wiki.js:
![image](https://user-images.githubusercontent.com/5561521/75978865-b6741400-5edf-11ea-8f7f-85da4e4afa84.png)

---

**Host Info:**
 - OS: Ubuntu 18.04, Docker
 - Wiki.js version: 2.1.114
 - Database engine: postgres 11.6

**Additional context**
I have no idea how the KaTeX module works but could I be missing some fonts locally perhaps or would it be related to how the module is implemented in wiki.js?

**Comment by NGPixel** [comment]
You need to disable **HTML Sanitization** under Admin > Rendering > HTML > Security, then re-save the page again.

I'm still investigating why the sanitizer breaks katex but disabling the option should fix your issue.

**Comment by NGPixel** [comment]
Fixed by 830f51664c33daa7b79d58ee06af1a57abc7db06
